talentmap_api/fsbid/views/bid_audit.py

```diff
@@ -233,3 +233,35 @@ def post(self, request):
             return Response(status=status.HTTP_404_NOT_FOUND)
 
         return Response(result)
+
+
+class FSBidBidAuditDeleteGradeListView(APIView):
+    '''
+    Delete existing At Grade relationship
+    '''
+
+    def post(self, request):
```

Comment:
Shouldn't we follow REST a tiny bit and set this as:

```suggestion
    def delete(self, request):
```

even though backoffice uses post we can still make our API have the regular put post get delete etc.